### PR TITLE
Meetings are now inside

### PR DESCRIPTION
**Pull request by howenc**
Update 2.10.2023 1430
image
image
image
image
image

1. Meetings will be stored like this
2. Attendees are not considered when finding duplicates
3. You can add as much attendees in a meeting
4. currently only way to view is from the data storage

### CODE REVIEW

**Comment by howenc** [comment]
Add link between attendee and Person, so that we can actually see a 1 to 1 relationship.

File: src/main/java/seedu/address/logic/parser/AddMeetingCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+        argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_TITLE, PREFIX_LOCATION, PREFIX_START, PREFIX_END);
+        Title title = ParserUtil.parseTitle(argMultimap.getValue(PREFIX_TITLE).get());
+        Location location = ParserUtil.parseLocation(argMultimap.getValue(PREFIX_LOCATION).get());
+        LocalDateTime start = ParserUtil.parseTime(argMultimap.getValue(PREFIX_START).get());
+        LocalDateTime end = ParserUtil.parseTime(argMultimap.getValue(PREFIX_END).get());
+        if (!MeetingTime.isValidMeetingTime(start, end)) {
+            throw new ParseException(MeetingTime.MESSAGE_CONSTRAINTS);
+        }
+        Set<Attendee> attendeeList = ParserUtil.parseAttendees(argMultimap.getAllValues(PREFIX_NAME));
```
Comment (jason-raiin) [inline src/main/java/seedu/address/logic/parser/AddMeetingCommandParser.java:49]: remove the functionality to add attendees from the add meetings command for now

File: src/main/java/seedu/address/logic/commands/AddMeetingCommand.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        requireNonNull(model);
+
+        if (model.hasMeeting(toAdd)) {
+            throw new CommandException(MESSAGE_DUPLICATE_MEETING);
+        }
+        for (Attendee attendee : toAdd.getAttendees()) {
+            if (!model.hasName(attendee.getAttendeeName())) {
+                throw new CommandException(MESSAGE_ATTENDEE_NOT_FOUND);
+            }
+        }
```
Comment (jason-raiin) [inline src/main/java/seedu/address/logic/commands/AddMeetingCommand.java:64]: remove the functionality to add attendees from the add meetings command for now

File: src/main/java/seedu/address/logic/commands/AddMeetingCommand.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+            + PREFIX_START + "START "
+            + PREFIX_END + "END "
+            + "[" + PREFIX_NAME + "ATTENDEES]...\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_TITLE + "Meeting with friends "
+            + PREFIX_LOCATION + "Starbucks "
+            + PREFIX_START + "18.10.2023 1200 "
+            + PREFIX_END + "18.10.2023 1400 "
+            + PREFIX_NAME + "John Doe "
+            + PREFIX_NAME + "Dohn Joe";
```
Comment (jason-raiin) [inline src/main/java/seedu/address/logic/commands/AddMeetingCommand.java:37]: remove the functionality to add attendees from the add meetings command for now

File: src/main/java/seedu/address/model/person/Name.java [hunk 4]
```diff
@@ -13,7 +13,7 @@ public class Name {
             "Names should only contain alphanumeric characters and spaces, and it should not be blank";
 
     /*
-     * The first character of the address must not be a whitespace,
+     * The first character of the title must not be a whitespace,
```
Comment (jason-raiin) [inline src/main/java/seedu/address/model/person/Name.java:16]: this should be name right?

**Review by jason-raiin — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/model/person/Name.java (hunk 4 above)
Comment (howenc) [reply src/main/java/seedu/address/model/person/Name.java:16]: Yes, it just has to be a string.

File: src/main/java/seedu/address/logic/commands/AddMeetingCommand.java (hunk 2 above)
Comment (howenc) [reply src/main/java/seedu/address/logic/commands/AddMeetingCommand.java:61]: I removed the surface level functionality, the addm command will no longer put any names into the attendee list.

File: src/main/java/seedu/address/logic/parser/AddMeetingCommandParser.java (hunk 1 above)
Comment (howenc) [reply src/main/java/seedu/address/logic/parser/AddMeetingCommandParser.java:49]: I removed the surface level functionality, the addm command will no longer put any names into the attendee list.

File: src/main/java/seedu/address/model/person/Name.java (hunk 4 above)
Comment (jason-raiin) [reply src/main/java/seedu/address/model/person/Name.java:16]: as in, the comment should say 'name' instead of 'title' right?

**Comment by jason-raiin** [comment]
closes #7

File: src/main/java/seedu/address/model/person/Name.java (hunk 4 above)
Comment (howenc) [reply src/main/java/seedu/address/model/person/Name.java:16]: Ah crap.

**Review by jason-raiin — APPROVED** [review]
lgtm

**Pull request merged**